pkg/services/sqlstore/migrations/ualert/alert_rule.go

```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"time"
 
+	ngmodels "github.com/grafana/grafana/pkg/services/ngalert/models"
```

Comment:
Can you please group this dependency with the ones at the bottom that relate to grafana - I'm not sure why (or when) `gofmt` stopped doing that.

Reply:
I don't think `gofmt` sorts imports that are separated with two or newlines